secure_message/resources/threads.py

```diff
@@ -17,12 +17,13 @@ class ThreadById(Resource):
     @staticmethod
     def get(thread_id):
         """Get messages by thread id"""
-        string_query_args, page, limit, ru_id, survey, cc, label, desc, ce = get_options(request.args)
+        string_query_args, page, limit, _, _1, _2, _3, _4, _5 = get_options(request.args)  # NOQA TODO - Named Tuple
```

Comment:
Same here with `_` unless there's a specific name for reference, may as well have it as `_`

Reply:
see above